app/addons/activetasks/stores.js

```diff
@@ -0,0 +1,127 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  "app/helpers",
+  "api",
+  "addons/activetasks/actiontypes"
+], function (Helpers, FauxtonAPI, ActionTypes) {
+
+  var ActiveTasksStore = FauxtonAPI.Store.extend({  
+    //elements in here
+
+    getSelectedTab: function () {
+      return this._selectedTab;
+    },
+    setSelectedTab: function (selectedTab) {
+      this._selectedTab = selectedTab;
+    },
+    getPollingInterval: function () {
+      return this._pollingInterval;
+    },
+    setPollingInterval: function (pollingInterval) {
+      this._pollingInterval = pollingInterval;
+    },
```

Comment:
Hi @kxepal! 

Not sure what you mean. Do you mean, for example, to create a `Polling` object, then define getters/setters on that object instead of attaching them to ActiveTaskStore object? Then likewise for the rest or the get/set functions here? that would make things a lot clearer :)